README.md

```diff
@@ -16,83 +16,41 @@ If something does not work, check the log file at `nextcloud/data/nextcloud.log`
 **⚠⚠ Warning:** If you are using more than one backend or especially one backend more often than once, make sure that you still have resp. get unique `uid`s in the database. ⚠⚠
 
 
-FTP
----
-Authenticate Nextcloud users against a FTP server.
-
-
-### Configuration
-You only need to supply the FTP host name or IP.
-
-The second - optional - parameter determines if SSL should be used or not.
-
-Add the following to `config.php`:
-
-    'user_backends' => array(
-        array(
-            'class' => 'OC_User_FTP',
-            'arguments' => array('127.0.0.1'),
-        ),
-    ),
-
-To enable SSL connections via `ftps`, append a second parameter `true`:
-
-    'user_backends' => array(
-        array(
-            'class' => 'OC_User_FTP',
-            'arguments' => array('127.0.0.1', true),
-        ),
-    ),
-
-
-### Dependencies
-PHP automatically contains basic FTP support.
-
-For SSL-secured FTP connections via ftps, the PHP [openssl extension][FTP_0]
-needs to be activated.
-
-[FTP_0]: http://php.net/openssl
-
-
-
 IMAP
 ----
 Authenticate Nextcloud users against an IMAP server.
 IMAP user and password need to be given for the Nextcloud login.
 
 
 ### Configuration
-The parameters are `host, port, sslmode, domain`.
-Possible values for sslmode are `ssl` or `tls`.
 Add the following to your `config.php`:
 
-    'user_backends' => array(
-        array(
+    'user_backends' => array (
+        0 => array (
             'class' => 'OC_User_IMAP',
-            'arguments' => array(
-                '127.0.0.1', 993, 'ssl', 'example.com', true, false
+            'arguments' => array (
+                0 => '127.0.0.1',
+                1 => 993,
+                2 => true,
+                3 => 'example.com',
+                4 => true,
+                5 => false,
             ),
         ),
     ),
 
-This connects to the IMAP server on IP `127.0.0.1`.
-The default port is 143. However, note that parameter order matters and if
-you want to restrict the domain (4th parameter), you need to also specify
-the port (2nd parameter) and sslmode (3rd parameter; set to `null` for
-insecure connection).
-If a domain name (e.g. example.com) is specified, then this makes sure that
-only users from this domain will be allowed to login. If the fifth parameter
-is set to true, after successfull login the domain part will be striped and
-the rest used as username in Nextcloud. e.g. 'username@example.com' will be
-'username' in Nextcloud. The sixth parameter toggles whether on creation of
-the user, it is added to a group corresponding to the name of the domain part
-of the address. 
+`0`: IMAP server  
+`1`: IMAP server port (default `143`)  
+`2`: secure connection  
```

Comment:
okay, update for this parameter (also for the example above). We missed the case of starttls (as in #135). So code changes under way in #138. To conclude there is:
- `ssl` for ssl/tls connections as normaly on port 993 which are encrypted from the beginning
- `tls` for STARTTLS which is normally used on port 143 (so connection is initialized but then "upgraded" to a tls connection via a respective IMAP command with instructions)
- `notls` (or technically any different value as of the implementations, but `notls` should be used) for plain connections (normally on port 143)

Reply:
okay, this might change again, as there is an intention to default to STARTTLS (everything that's not explicit ssl or notls) in #140